test/routes/events/browse.js

```diff
@@ -17,22 +17,33 @@ const { expect } = require('code');
 const lab = exports.lab = require('lab').script();
 const url = require('url');
 
-const server = require('../../../');
-const { destroyRecords, getAuthToken, fixtures } = require('../../fixture-client');
-const { users, events } = require('../../fixtures');
+const server = require('../../..');
+const { createMapRelations, destroyRecords, getAuthToken } = require('../../fixture-client');
```

Comment:
@duncanbeevers fyi i got a bit confused and included fixtures here instead of as below with ../../fixtures and {events,users} = fixtures; -- it still seems to work but if i've mangled one of these include sections in my merge commit please check me and I apologize